docs/reference/scripting/using.asciidoc

```diff
@@ -70,7 +70,7 @@ GET my-index-000001/_search
   "script_fields": {
     "my_doubled_field": {
       "script": { <1>
-        "source": "doc['my_field'].value * params['multiplier']", <2>
+        "source": "field('my_field').get(null) * params['multiplier']", <2>
```

Comment:
What do you think about using `0` as a default?

Reply:
For primitive types (numbers and boolean), null cannot be used as the default, so it has to be some number -- @stu-elastic's suggestion of `0` makes sense to me.

Reply:
I'll change `null` to `0` here